internal/configs/virtualserver.go

```diff
@@ -908,10 +907,32 @@ func (p *policiesCfg) addIngressMTLSConfig(
 		verifyClient = ingressMTLS.VerifyClient
 	}
 
-	p.IngressMTLS = &version2.IngressMTLS{
-		ClientCert:   secretRef.Path,
-		VerifyClient: verifyClient,
-		VerifyDepth:  verifyDepth,
+	caFields := strings.Fields(secretRef.Path)
+
+	if _, hasCrlKey := secretRef.Secret.Data[CACrlKey]; hasCrlKey && ingressMTLS.Crl != "" {
+		res.addWarningf("Both ca.crl and ingressMTLS.crl fields cannot be used. ca.crl will be ignored and %s will be applied", polKey)
```

Comment:
I would be more explicit here that the ca.crl is coming from the secret, something like
```suggestion
		res.addWarningf("Both ca.crl in the Secret and ingressMTLS.crl fields cannot be used. ca.crl in %s will be ignored and %s will be applied", secretKey, polKey)
```